baker/siteRenderers.tsx

```diff
@@ -256,103 +253,27 @@ export const renderPost = async (
     )
 }
 
-// This function is temporarily forked until we have fully transitioned to a gdocs homepage,
-// whereupon we'll strip out the old front page code.
 export const renderFrontPage = async (knex: Knex<any, any[]>) => {
-    // Annoying, MySQL+TypeORM doesn't support JSONB, so I'm using raw SQL to confirm if there's a published homepage
-    const gdocHomepageResult = await db.knexRawFirst<{ id: string }>(
-        `SELECT id FROM posts_gdocs WHERE content->>"$.type" = "${OwidGdocType.Homepage}" AND published = TRUE`,
-        db.knexInstance()
+    const gdocHomepageResult = await knex.raw<{ id: string }>(
+        `--sql
+        SELECT id
+        FROM posts_gdocs
+        WHERE content->>"$.type" = "${OwidGdocType.Homepage}"
+        AND published = TRUE`
     )
```

Comment:
We probably need to revert this change:
- the "--" needs a space after it (otherwise I get a syntax error)
- gdocHomepageResult is an array so gdocHomepageResult.id below doesn't return what we would expect

Reply:
Wow yeah this was completely broken. Rewrote it and put the query in `db.ts`